src/logic/operate.js

```diff
@@ -0,0 +1,23 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let res;
+  const a = new Big(parseFloat(numberOne));
+  const b = parseFloat(numberTwo);
```

Comment:
- [ ] Also wrap the variable `b` in a new Big.js notation.

Reply:
Big.js docs don't require the second variable to be wrapped in Big.js , pls. see docs [example](http://mikemcl.github.io/big.js/#minus)